setup.py

```diff
@@ -27,7 +27,7 @@
     package_data={"fourmat": ("assets/*.*", "assets/.*")},
     install_requires=(
         "click>=7",
-        "black==21.11b1",
+        "black>=22,<23",
```

Comment:
Since black is "official" now, changing to use a range. (assuming black is versioned accordingly)

Reply:
Ah ok, that's why we pinned it I guess.

```suggestion
        "black==22.1.0",
```